### PR TITLE
BTAT-7641 Updating 'what you owe' content

### PR DESCRIPTION
**Pull request by paulfirmstone**
Test Data to use:
Direct Debit VRN = 555555555
No Direct Debit VRN = 666666666
API Failure VRN = 777777777
No Payments VRN = 888888888

### CODE REVIEW

File: app/views/payments/openPayments.scala.html [hunk 1]
```diff
@@ -55,20 +44,18 @@ <h1 class="heading-xlarge">@messages("openPayments.heading")</h1>
...
+            <p>@messages("openPayments.processingTime")</p>
+        </div>
+
+        <h2 id="what-you-owe-missing" class="heading-small">@messages("openPayments.whatOweHelp.heading")</h2>
+            <p>@messages("openPayments.incorrect")
+                <a target="_blank" href="https://www.gov.uk/vat-corrections">@messages("openPayments.correctErrors")</a>.
+            </p>
+            <p>@messages("payment.afterSubmitted") @Messages("payment.stillMake")
+                <a target="_blank" href="@appConfig.unauthenticatedPaymentsUrl" id="vatPaymentsLink">
+                    @messages("payment.stillMakeLink")<span class="visuallyhidden"> @messages("payment.evenIfNotShown")</span></a>.
```
Comment (sraspin) [inline app/views/payments/openPayments.scala.html]: Where did the hidden content come from? I cant see it in the content spreadsheet

File: app/views/payments/openPayments.scala.html [hunk 2]
```diff
@@ -55,20 +44,18 @@ <h1 class="heading-xlarge">@messages("openPayments.heading")</h1>
...
+            </p>
```
Comment (sraspin) [inline app/views/payments/openPayments.scala.html]: A bit nit-picky, but could you re-format this section of code please

File: app/views/payments/openPayments.scala.html (hunk 1 above)
Comment (paulfirmstone) [reply app/views/payments/openPayments.scala.html]: This was existing content and has been unchanged.

File: app/views/payments/openPayments.scala.html (hunk 2 above)
Comment (paulfirmstone) [reply app/views/payments/openPayments.scala.html]: Done

**Pull request merged**